announcer.py

```diff
@@ -36,7 +36,7 @@ def announce(self):
                 if self.slacker.channel_exists(config.announce_channel):
                     self.sb.say(config.announce_channel, m)
                 else:
-                    self.ds.warning("Attempted to announce in {}, but channel does not exist.".format(config.announce_channel))
+                    self.ds.logger.warning("Attempted to announce in %s, but channel does not exist.", config.announce_channel)
```

Comment:
Ah, interesting. I was following [the `logging` docs](https://docs.python.org/2/library/logging.html), which seem to have retained the `%s` format strings when values are being passed directly to a `Logger`.

That said, we could standardize on passing a fully `.format()`-ted string to all logger calls. I’m not sure if there are benefits or drawbacks to that vs. passing the values-to-be-formatted directly to the logger.